consensus/service/src/api/client_api_service.rs

```diff
@@ -187,6 +194,25 @@ mod client_api_tests {
         (client, server)
     }
 
+    /// Get a dummy config object
+    fn get_config() -> Config {
+        Config::try_parse_from(&[
+            "foo",
+            "--peer-responder-id=localhost:8081",
+            "--client-responder-id=localhost:3223",
+            "--msg-signer-key=MC4CAQAwBQYDK2VwBCIEIC50QXQll2Y9qxztvmsUgcBBIxkmk7EQjxzQTa926bKo",
+            "--network=network.toml",
+            "--peer-listen-uri=insecure-mcp://0.0.0.0:8081/",
+            "--client-listen-uri=insecure-mc://0.0.0.0:3223/",
+            "--admin-listen-uri=insecure-mca://0.0.0.0:9090/",
+            "--sealed-block-signing-key=/tmp/key",
+            "--ledger-path=/tmp/ledger",
+            "--ias-spid=22222222222222222222222222222222",
+            "--ias-api-key=asdf",
```

Comment:
should not we have here an entry for `block_version` which is used above on line 138? or that's a default value coming form somewhere?

Reply:
so what i'm doing here is actually passing an array of string literals which the `clap` code parses the same way it would parse `argc, argv`. So anything that is defaulted in the struct will be defaulted here also, I only had to list the required parameters to start the server

Reply:
i did it that way because it was less complicated than doing `Config { ... } ` and actually initializing everything

Reply:
and yeah i think block-version is defaulting to zero for backwards compat